### PR TITLE
[hail] add lambda GC calculation

### PR DESCRIPTION
**Pull request by konradjk**
Also added `table_source` since I couldn't find it - let me know if I missed it

### CODE REVIEW

File: hail/python/hail/methods/statgen.py [hunk 1]
```diff
@@ -1323,6 +1323,34 @@ def skat(key_expr, weight_expr, y, x, covariates, logistic=False,
...
+        Genomic inflation factor (lambda genomic control).
+    """
+    check_row_indexed('lambda_gc', p_value)
+    t = table_source(p_value)
+    chisq = hl.qchisqtail(p_value, 1)
+    if approximate:
+        med_chisq = hl.agg.approx_quantiles(chisq, 0.5)
+    else:
+        med_chisq = hl.median(hl.agg.collect(chisq))
+    return t.aggregate(med_chisq / hl.qchisqtail(0.5, 1))
```
Comment (tpoterba) [inline hail/python/hail/methods/statgen.py]: can we separate the bit that generates the agg expression out into a separate private function? We have an open feature request to print the lambda GC in `hl.plot.qq`

**Comment by tpoterba** [comment]
add to genetics.rst

**Comment by tpoterba** [comment]
also add experimental note since this is untested

File: hail/python/hail/methods/statgen.py [hunk 2]
```diff
@@ -1323,6 +1323,45 @@ def skat(key_expr, weight_expr, y, x, covariates, logistic=False,
...
 
+@typecheck(p_value=expr_numeric,
+           approximate=bool)
+def lambda_gc(p_value, approximate=True):
+    """
+    Compute genomic inflation factor (lambda GC) from an Expression of p-values.
+
+    Note
+    ----
+    This method is experimental.
```
Comment (tpoterba) [inline hail/python/hail/methods/statgen.py]: oops sorry this is what I meant:

```
    .. include:: ../_templates/experimental.rst
```

Comment (konradjk) [reply hail/python/hail/methods/statgen.py]: nifty!

**Pull request merged**